entropylab/results/dashboard/auto_plot.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+
+import numpy as np
+import plotly.graph_objects as go
+import plotly.express as px
+
+from entropylab.api.errors import EntropyError
+from entropylab.api.plot import CirclePlotGenerator
+
+
+def auto_plot(data) -> go.Figure:
```

Comment:
please make this function not implementation aware, so it returns the data and generator and plotting done outside

Reply:
I'll move it to the `DashboardDataReader` as originally planned.

Reply:
Done. Kindly review in `dashboard_data.py`.